cli/src/action/circuit/template.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::ArgMatches;
+
+use crate::error::CliError;
+use crate::template::CircuitTemplate;
+
+use super::Action;
+
+pub struct ListCircuitTemplates;
+
+impl Action for ListCircuitTemplates {
+    fn run<'a>(&mut self, _: Option<&ArgMatches<'a>>) -> Result<(), CliError> {
+        let templates = CircuitTemplate::list_available_templates()?;
+
+        println!("Available templates: {}", templates.join(", "));
```

Comment:
I think it would be easier to read if each template was printed on a new line

Reply:
updated